apps/vc-api/test/wallet-client.ts

```diff
@@ -155,9 +155,8 @@ export class WalletClient {
     const continueExchangeResponse = await request(this.#app.getHttpServer())
       .get(`${API_DEFAULT_VERSION_PREFIX}/vc-api/exchanges/${exchangeId}/${transactionId}`)
       .expect(200);
-    expect(continueExchangeResponse.body.errors).toHaveLength(0);
-    expect(continueExchangeResponse.body.transaction).toBeDefined();
```

Comment:
What's your opinion having an assertion on the request result in this method? I think at the time when I was writing it, I thought that having the shared assertion prevented tests that calling this method from forgetting about it and so I thought it wouldn't hurt. But at the same time, I feel as though maybe having assertions hidden in helper methods could be confusing to a test reader.

What are your thoughts @artursudnik ?

Reply:
@jrhender 
Asserting errors is not necessary now, because in case of errors it will not respond with 200. When the transaction is not found it also responds with a 404 status code: https://github.com/energywebfoundation/ssi/blob/2049034aa3ba94fe9770dcf860cb47304a83c542/apps/vc-api/src/vc-api/vc-api.controller.ts#L274-L274